code/modules/spells/aoe_turf/conjure/pitbulls.dm

```diff
@@ -18,12 +18,33 @@
 	hud_state = "pitbull"
 	cast_sound = 'sound/voice/pitbullbark.ogg'
 	quicken_price = Sp_BASE_PRICE
+	var/empowered
+
+/spell/aoe_turf/conjure/pitbull/empower_spell()
+	..()
+	empowered += 1
+	spell_levels[Sp_POWER]++
+	. = "You have perfected the SMASHED and SLAMMED summon."
+
+/spell/aoe_turf/conjure/pitbull/invocation(mob/user, list/targets)
+	if(empowered)
+		invocation = pick("P'MPY S'N 'PP", "P'MPY S'N 'PP", "R'V'R'D' K'NN'LS", "BL'DSK'LL") 
+	..()
 
 var/list/pitbulls_exclude_kinlist = list() //all pitbulls go in here so pitbulls won't attack other pitbulls when feeling treacherous (and instead attack the wizard)
 
+/spell/aoe_turf/conjure/pitbull/perform(mob/user = usr, skipcharge = 0, list/target_override, var/ignore_timeless = FALSE, var/ignore_path = null)
```

Comment:
not sure if this specific change will break stuff cause it didn't cause any issues as a plain perform() call on local, but it failed the linter check here without all that stuff in the (), but I can't test that today